packages/docs/pages/reference/elements/ol.mdx

```diff
@@ -8,12 +8,64 @@ import {
 
 # `<ol>`
 
-Short description of what the `<ol>` tag does.
+The `<ol>` tag renders an ordered list with default markers which can (if
+desired) be modified.  For comparison, see [`<ul>`](./ul).
 
 ## Syntax
 
 <SyntaxDisplay name="ol" />
```

Comment:
It would be good to list all the possibilities for `marker`. This can be done by changing this line to
```jsx
<SyntaxDisplay
    name="ol"
    defaultVariant="Ol"
    additionalAttributeInfo={{
        Ol: { marker: { value: "(1) (A) (a) (I) (i)" } }
    }}
/>
```
This also makes it so that the `<ol>` for a list in a paragraph is the default that shows, rather than the `<ol>` exercises.